docs/customization.md

```diff
@@ -407,6 +407,29 @@ A comma delimited list of roles (minus the role/ prefix) that can be used when c
 
 
 
+## Compatibility
+
+Legacy Compatibility Options
+
+You can configure the following environment variables:
+
+<b><tt>GSB_COMPATIBILITY_THREE_TO_FOUR</tt></b> - <i>boolean</i> - Compatibility with GCP Service Broker v3.X
```

Comment:
This variable name assumes responsibility for all compatibility issues between 3 and 4.

Just double checking that we're ok with that. If say, we find more issues and we use this name, this feature flag will be an all or nothing compatibility layer.

It might be better to scope the feature flag to something like `GSB_THREE_TO_FOUR_PLAN_COMPATIBILITY`

Or even `GSB_LEGACY_PLAN_COMPATIBILITY`

Reply:
Good catch, I'll update this to `GSB_COMPATIBILITY_THREE_TO_FOUR_LEGACY_PLANS` with the internal property name being `compatibility.three-to-four.legacy-plans` to keep the general to specific property name flow.